snippets/variant-list-row.liquid

```diff
@@ -0,0 +1,233 @@
+{% comment %}
+  Variant List Row
+
+  Accepts:
+  - item: {Object} Variant or Product object
+  - sku: {String} Sku of Product or Variant (optional)
+  - image: {Object} Product or Variant image (optional)
+  Usage:
+  {% render 'variant-list-row' variant: variant %}
+{% endcomment %}
+
+<tr class="variant-item" id="Variant-{{ variant.id }}">
+  <td class="variant-item__contain">
```

Comment:
`__contain` is a new pattern and not very clear. I'd actually think this class would wrap all the cells or is at least used as a generic container in each column. Could we choose a more informative name? Maybe using "media", "label", or "title"

Reply:
`media` is already the one that wraps the image below it and `title` didn't make sense to me because i show the name again below in a different screensize. What about `variant-item__content`?